actions/entities/delete.php

```diff
@@ -6,17 +6,6 @@
  * @subpackage Core
  */
 
-$guid = get_input('guid');
-$entity = get_entity($guid);
+elgg_deprecated_notice('entities/delete action file has been deprecated. Use delete action instead', '2.1');
 
-if (($entity) && ($entity->canEdit())) {
-	if ($entity->delete()) {
-		system_message(elgg_echo('entity:delete:success', array($guid)));
-	} else {
-		register_error(elgg_echo('entity:delete:fail', array($guid)));
-	}
-} else {
-	register_error(elgg_echo('entity:delete:fail', array($guid)));
-}
-
-forward(REFERER);
+include_once dirname(dirname(__FILE__)) . '/delete.php';
```

Comment:
```php
return (require __DIR__ . '/../delete.php');
```
We don't do anything with return values, but we might as well pass it through in case we start.